src/replication.c

```diff
@@ -3193,7 +3186,7 @@ void setupMainConnForPsync(connection *conn) {
     if (psync_result == PSYNC_WAIT_REPLY) return; /* Try again later... */
 
     if (psync_result == PSYNC_CONTINUE) {
-        serverLog(LL_NOTICE, "Primary <-> REPLICA sync: Primary accepted a Partial Resynchronization%s",
+        serverLog(LL_NOTICE, "Primary <-> REPLICA sync: Primary accepted a partial re-synchronization%s",
```

Comment:
Capitalization is inconsistent in "Primary <-> REPLICA". In the unstable branch it seems to be uppercase "PRIMARY <-> REPLICA". Is this a merge conflict solving mistake?

Reply:
This wasn't a matter of resolving a conflict, I just wasn't familiar with that standard. Will fix.